base58/__init__.py

```diff
@@ -93,7 +102,7 @@ def b58decode_int(
 
 
 def b58decode(
-    v: Union[str, bytes], alphabet: bytes = BITCOIN_ALPHABET
+    v: Union[str, bytes], alphabet: bytes = BITCOIN_ALPHABET, autofix: bool = False
```

Comment:
I think this should be added as a keyword only argument by adding ', *, ' to the signature here and in b56decode_int

Reply:
I'm not sure how to do it. Could you please offer your patch?

Reply:
```suggestion
    v: Union[str, bytes], alphabet: bytes = BITCOIN_ALPHABET, *, autofix: bool = False
```